ruby/spec/session_spec.rb

```diff
@@ -72,12 +72,10 @@ def expect_request(body: nil, endpoint: nil)
         expect(session.data_center).to eq :US_WEST
       end
 
-      it 'uses provided Data Center, Username, Access Key' do
-        session = Session.new(data_center: :EU_VDC,
-                              username: 'bar',
-                              access_key: '321')
+      it 'uses provided Data Center' do
+        session = Session.new(data_center: :EU_VDC,)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid comma after the last parameter of a method call.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4779388)